runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -199,6 +199,7 @@
                   <includes>
                     <include>com.google.guava:guava</include>
                     <include>com.google.protobuf:protobuf-java</include>
```

Comment:
sdks/java/core, runners/direct-java should be updated as well if it was found that protobuf-java depends on having protobuf-java-util

Also, update gradles shade configurations.

Reply:
Done. They aren't exactly consistent, so I kept each thing internally consistent. Wherever protobuf is shaded, I shaded protobuf-util.

To be clear, protobuf-java-util is used by `WindowingStrategyTranslation`, so it is places that use both protobuf-java-util and also translation aka runners/core-construction-java. So we don't always have to shade them both together.